weblate/addons/json.py

```diff
@@ -41,3 +41,7 @@ def store_post_load(self, translation, store) -> None:
         else:
             store.store.dump_args["indent"] = "\t" * indent
         store.store.dump_args["sort_keys"] = bool(int(config.get("sort_keys", 0)))
+        use_compact_separators = bool(int(config.get("use_compact_separators", 0)))
+        store.store.dump_args["separators"] = (
+            (",", ":") if use_compact_separators else (", ", ": ")
```

Comment:
As indent is always turned on, this should be:

```suggestion
            (",", ":" if use_compact_separators else ": ")
```

See https://docs.python.org/3/library/json.html#json.dump

The tests I've added in https://github.com/WeblateOrg/weblate/pull/14795 reveal this.